### PR TITLE
Feature/game ag

### PR DESCRIPTION
**Pull request by algibson1**
Create Game class and tests

### CODE REVIEW

File: lib/game.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    @home_goals = game_info[:home_goals].to_i
+  end
+  #StatTracker will reference total_goals for highest_total_score and lowest_total_score
+  def total_goals
+    @away_goals + @home_goals
+  end
+  # To use in highest_scoring_visitor, highest_scoring_home_team, etc? Or else other class using the game_team file data?
+  def winner
+    return "home" if @home_goals > @away_goals
+    return "away" if @away_goals > @home_goals
```
Comment (dani-wilson) [inline lib/game.rb:25]: I think this method is a winner! 🥁

File: spec/game_spec.rb [hunk 2]
```diff
@@ -15,21 +15,45 @@
...
     @game_4 = games[5] 
     @game_5 = games[12]
+    @game_6 = games[9]
   end
 
   it "exists" do
     expect(@game_1).to be_a(Game)
   end
 
-  it "has game_id, season, season_type, away_team_id, home_team_id, away_goals, home_goals" do
```
Comment (DChavez18) [inline spec/game_spec.rb]: I'm wondering if we should put this into a describe block for readability?

Thank you for separating out the tests!

File: spec/game_spec.rb [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    expect(@game_2.home_team_id).to eq("3")
+    expect(@game_3.away_team_id).to eq("29")
+    expect(@game_3.home_team_id).to eq("24")
+  end
+
+  it "has goals" do
+    expect(@game_1.away_goals).to eq(2)
+    expect(@game_1.home_goals).to eq(3)
+    expect(@game_3.away_goals).to eq(2)
+    expect(@game_3.home_goals).to eq(3)
```
Comment (dani-wilson) [inline spec/game_spec.rb:54]: These tests look real robust

**Comment by dani-wilson** [comment]
Your commits look great. 
Reviewed & commented on by Dani @ 8:50 pm MST

**Review by DChavez18 — APPROVED** [review]

File: spec/game_spec.rb (hunk 2 above)
Comment (DChavez18) [reply spec/game_spec.rb]: Referring to the attribute tests

Comment (algibson1) [reply spec/game_spec.rb]: I like that idea! If we do describe blocks in this spec, we should do them in all specs for consistency. I suggest adding this to our Miro board for our "when we have time" category so we can check that all our specs have describe blocks

File: lib/game.rb [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+              :home_team_id,
+              :away_goals,
+              :home_goals
+  def initialize(game_info)
+    @game_id = game_info[:game_id]
+    @season = game_info[:season]
+    @season_type = game_info[:type]
+    @away_team_id = game_info[:away_team_id]
+    @home_team_id = game_info[:home_team_id]
+    @away_goals = game_info[:away_goals].to_i
```
Comment (GVarenhorst) [inline lib/game.rb:15]: good catch! I remember this throwing  us off for a bit with my previous team.

**Review by dani-wilson — APPROVED** [review]

File: spec/game_spec.rb [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+  it "has a total goals scored" do
+    expect(@game_1.total_goals).to eq(5)
+    expect(@game_2.total_goals).to eq(3)
+    expect(@game_3.total_goals).to eq(5)
+    expect(@game_4.total_goals).to eq(3)
+    expect(@game_5.total_goals).to eq(6)
+  end
+
+  it "has a winner" do
```
Comment (GVarenhorst) [inline spec/game_spec.rb:67]: Would it make sense at this point to test "loser" as well using your winner method?

**Review by GVarenhorst — CHANGES_REQUESTED** [review]
With regards to winner method, could we also test loser? Basically a !winner spec test?

**Comment by algibson1** [comment]
Closing to reopen a new PR with updated branch

**Pull request closed without merging**